### ISSUE TITLE
Report in status bar when wallet is in stuck mode.

### ISSUE THREAD

**Issue by Transisto** [sparrowwallet/sparrow#439]
The colour up there change to red but I don't think it's obvious enough.

![image](https://user-images.githubusercontent.com/388716/155608531-a0ef9ecb-c7bd-4c50-874e-00e2b5889546.png)

[Feb 24 11:50:43] Failed to subscribe to path: ../0/762

Actually, I think it should just not stop this often.

Currently Sparrow needs to be restarted every day to be stable on OSX M1 chip.

**Comment by craigraw** [comment]
Yes, I think this is an area that needs more thought - the red icon gets "what is the meaning of this" type questions.

> Actually, I think it should just not stop this often.

Unsure what you mean by this? It's a server issue (no response), not Sparrow.

> Currently Sparrow needs to be restarted every day to be stable on OSX M1 chip.

Planning to acquire the hardware to test on this platform soon. Let's discuss on the related issue tho.

**Comment by hur4hur5** [comment]
I also have been confused by the red folder icon. I presume red means bad but beyond that I haven't a clue. Is it intermittent bad? When a user sees this for the first time should they panic or worry? 

The text to the right of the folder icon within the tabs at the top of the application is the name of the wallet within Sparrow. Maybe have the text in the tabs cycle between the name of the wallet and a brief error description whilst the folder icon is red? For example "Wallet Name" -> "Error Message" -> "Wallet Name".

**Comment by brendanphall** [comment]
Telegram notification options from sparrow, including the 'red wallet' icon notification would be appreciated.

**Comment by Transisto** [comment]
> Telegram notification options from sparrow, including the 'red wallet' icon notification would be appreciated.

What do you mean by Telegram?  You mean having a popup notification when there's connection issues? Like when you received BTC?

**Comment by brendanphall** [comment]
Sure a notification, but through https://telegram.org/ so it could be received while away form that PC. I'm not monitoring the PC sparrow is running throughout the day. If there are connection issues, say 30min of 'red wallet' status, then a notification could be sent which could be received remotely and I could attend to it. As it stands now, I have to randomly check and find it in a disconnected state - lots of wasted downtime.
